src/model/userDetail.js

```
@@ -0,0 +1,50 @@
+const Sequelize = require("sequelize");
+
+class UserDetail extends Sequelize.Model {
+  static initiate(sequelize) {
+    super.init(
```

Biome (1.9.4)

[error] Using super in a static context can be confusing.

super refers to a parent class.

(lint/complexity/noThisInStatic)